factorial.py

```diff
@@ -0,0 +1,20 @@
+def factorial(n):
+    fact = 1
+    for i in range(1, n):
+        fact *= i
+    return fact
+
+def is_even(n):
+    if n % 2 = 0:
+        return True
```

Comment:
The `else` block in `is_even` is unnecessary. The function can directly return `n % 2 == 0`.